### PR TITLE
WorkflowManagerActorSpec fixup. Closes #1028

### LINKED ISSUE #1028: Assess/Fix/Implement Ignored Tests in WorkflowManagerActorSpec
The goal here is to look at the tests in the spec, and determine
1. does the test assess something valuable
2. if no, delete it (with an explanation in the PR)
3. if yes, either fix it up so it passes, or reimplement... whatever is more expedient

### PR DESCRIPTION
**Pull request by cjllanwarne**
Mainly this is moving tests for functionality which isn't there yet into different files. One test remains, and two placeholders

### CODE REVIEW

**Comment by cjllanwarne** [comment]
The wheel room is currently indisposed, so I've assigned this PR based on "who hasn't got a PR assigned to them yet"

File: engine/src/test/scala/cromwell/engine/workflow/MaterializeWorkflowDescriptorActorSpec.scala [hunk 1]
```diff
@@ -213,5 +213,15 @@ class MaterializeWorkflowDescriptorActorSpec
 //      }
 //      verify(backendMock, times(1)).assertWorkflowOptions(WorkflowOptions.fromJsonString(customizedLocalBackendOptions).get)
     }
+
+    // TODO: PBE These tests have migrated here from WorkflowManagerActorSpec:
+    "Handle coercion failures gracefully" ignore {
```
Comment (geoffjentry) [inline engine/src/test/scala/cromwell/engine/workflow/MaterializeWorkflowDescriptorActorSpec.scala]: Is this just to remind ourselves that they need to exist at some point?

**Comment by geoffjentry** [comment]
👍

**Comment by Horneth** [comment]
👍

File: engine/src/test/scala/cromwell/engine/workflow/MaterializeWorkflowDescriptorActorSpec.scala (hunk 1 above)
Comment (cjllanwarne) [inline engine/src/test/scala/cromwell/engine/workflow/MaterializeWorkflowDescriptorActorSpec.scala]: I took the ticket for MWDASpec already, so I might tidy this up into a "invalid coercions are invalid" test

**Pull request merged**